automation/test.sh

```diff
@@ -70,6 +70,15 @@ else
   export KUBEVIRT_PROVIDER=${TARGET}
 fi
 
+if [[ $TARGET =~ psa ]]; then
```

Comment:
@xpivarc I am not sure that it's going to work, since we use `-apply-default-e2e-configuration` flag in the `functests.sh` and it overrides the feature-gate list.

Reply:
@xpivarc Sorry my bad, we append. Also checked in the failed test artifacts.

Reply:
BTW. We have also test that verify this holds true in the future.